packages/cli/README.md

```diff
@@ -20,12 +20,12 @@ $ yarn add @backstage/cli
 
 For local development the cli can be used directly, even from other packages in this repo. The `bin/backstage-cli` entrypoint contains a switch that will load the implementation from the `src` directory when executed inside this repo.
 
-To run the cli in watch mode, use `yarn start <args>`. For example `yarn start create-app --help`.
+To run the cli in watch mode, use `yarn start <args>`. For example `yarn start lint --help`.
 
-To try out the `create-app` command locally, you can execute the following from the parent directory of this repo:
+To try out the `lint` command locally, you can execute the following from the parent directory of this repo:
 
 ```bash
-./backstage/packages/cli/bin/backstage-cli create-app
+./backstage/packages/cli/bin/backstage-cli lint
```

Comment:
Better use a different example than `lint`, since that currently needs to be run through a yarn script to have the correct `PATH` env set up. Maybe just `--help`?

Reply:
done